test/integration/targets/win_app_control/test_enabled.yml

```diff
@@ -0,0 +1,215 @@
+- name: run signed module
+  ns.col.signed:
+    input: café
+  register: signed_res
+
+- name: assert run signed module
+  assert:
+    that:
+    - signed_res.language_mode == 'FullLanguage'
+    - signed_res.test == 'signed'
+    - signed_res.whoami == current_user
+    - signed_res.ünicode == 'café'
+
+- name: run inline signed module
+  ns.col.inline_signed:
+    input: café
+  register: inline_signed_res
+
+- name: assert run inline signed module
+  assert:
+    that:
+    - inline_signed_res.language_mode == 'FullLanguage'
+    - inline_signed_res.test == 'inline_signed'
+    - inline_signed_res.whoami == current_user
+    - inline_signed_res.ünicode == 'café'
+
+- name: run inline signed module that is not trusted
+  ns.col.inline_signed_not_trusted:
+    input: café
+  register: inline_signed_not_trusted_res
+
+- name: assert run inline signed module
+  assert:
+    that:
+    - inline_signed_not_trusted_res.language_mode == 'ConstrainedLanguage'
+    - inline_signed_not_trusted_res.test == 'inline_signed_not_trusted'
+    - inline_signed_not_trusted_res.whoami == current_user
+    - inline_signed_not_trusted_res.ünicode == 'café'
+
+- name: run signed module to test exec wrapper scope
+  ns.col.scope:
+  register: scoped_res
+
+- name: assert run signed module to test exec wrapper scope
+  assert:
+    that:
+    - scoped_res.missing_using_namespace == True
+    - scoped_res.module_using_namespace == 'System.Management.Automation.Language.Parser'
+    - scoped_res.script_var == 'foo'
+
+- name: run module marked as skipped
+  ns.col.skipped:
+    input: café
+  register: skipped_res
+
+- name: assert run module marked as skipped
+  assert:
+    that:
+    - skipped_res.language_mode == 'ConstrainedLanguage'
+    - skipped_res.test == 'skipped'
+    - skipped_res.whoami == current_user
+    - skipped_res.ünicode == 'café'
+
+- name: run module marked as skipped with a failure
+  ns.col.skipped:
+    should_fail: true
+  register: skipped_fail_res
+  ignore_errors: true
+
+- name: assert run module marked as skipped with a failure
+  assert:
+    that:
+    - skipped_fail_res is failed
+    - >-
+      skipped_fail_res.msg == "Unhandled exception while executing module: exception here"
+    - skipped_fail_res.exception is search("At .*\\\\ansible_collections\.ns\.col\.plugins\.modules\.skipped-.*\.ps1")
+
+- name: run module marked as unsupported
+  ns.col.unsupported:
+  register: unsupported_res
+  failed_when:
+  - unsupported_res.failed == False
+  - >-
+    unsupported_res.msg != "failure during exec_wrapper: Provided script for 'ansible_collections.ns.col.plugins.modules.unsupported.ps1' is marked as unsupported in CLM mode."
```

Comment:
Just for consistency with most of the other revamped 2.19 error messages, you might want to consider tweaking the capitalization and text on the boilerplate part, like `Powershell exec_wrapper failed: ` (couldn't comment on the actual message source since it was in a previous changeset)

Reply:
Ditto will fix up to do more of a pattern match.